### PR TITLE
Move puppetlabs-kubernetes to working patched version

### PR DESCRIPTION
**Pull request by fydai**
Upstream PR is open, should be merged in the next release. Meanwhile this allows the kubernetes masters to be set back to production.

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -18,6 +18,7 @@ mod 'puppetlabs-firewall',             '1.12.0'
 mod 'puppetlabs-hocon',                '1.0.0'
 mod 'puppetlabs-inifile',              '2.2.2' # Dependency of puppetlabs-puppetdb
 mod 'puppetlabs-kubernetes',           '3.1.0'
+mod 'puppetlabs-kubernetes', :git => 'https://github.com/fydai/puppetlabs-kubernetes'
```
Comment (dkess) [inline Puppetfile]: Should the line above this be removed?

**Comment by fydai** [comment]
yes

File: Puppetfile (hunk 1 above)
Comment (Baisang) [reply Puppetfile]: You made your commit off of master of puppetlabs-kubernetes in your fork, so by doing this you would be upgrading the version we are using to their master branch at the same time.

You may want to consider rebasing your changes off of version 3.1.0 in your fork, tagging it as something else, and using that as the version.

**Comment by Baisang** [comment]
That being said, if it still works 100%, feel free to resolve the issue I brought up.

**Comment by fydai** [comment]
As my pull request has been merged, I have updated this pull request to use the newest version of puppetlabs-kubernetes, 4.0.1, which includes my changes.

**Pull request merged**